templates/birds/utility.macros.j2

```diff
@@ -5,12 +5,14 @@
     <div class="h-card">
       {{ utility.icon(page.config.bird, size='face') }}
 
-      {{ page.render_summary() }}
+      <div class="bio-short">
```

Comment:
I tried with with the long bio and while it looks OK on desktop, it was really tall on mobile. What if we made our short bios a tiny bit longer? 

Regardless, yes, a new story I think would be good.